### PR TITLE
I changed the classname from small to small-button

### PR DESCRIPTION
**Pull request by Poorooshka**
# Description

The buttons component started dysfunctioning after it was merged and pulled from Github. Since classname .small had conflict with another CSS file, it was replaced by small-button.
I also added a shadow to the button.

# How to test?
You can see it after npm run stroybook.

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by dpfernandes — COMMENTED** [review]
Why is this comment useful? 

//  "small" changed to "small-button" due to conflicts with another CSS file

**Review by Enqira — CHANGES_REQUESTED** [review]
Good job Nashmil, but could change a couple of things:
On  hover add mouse to pointer and background to a little different color
Also if you could remove the console log, I added a comment on that.

File: src/client/components/Buttons/Buttons.stories.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export const Button = Template.bind({});
 
 Button.args = {
   label: 'Click me',
-  size: 'small',
+  size: 'small-button',
   isMono: false,
   color: 'orange',
   onClick: () => {
```
[the remark this answers is not in the document]
Comment (Poorooshka) [reply src/client/components/Buttons/Buttons.stories.js]: I did

**Review by Enqira — APPROVED** [review]
Looks good, redy to be merged

**Comment by Enqira** [comment]
So Nashmil, Daniel pointed that you have a conflict in the Buttons.styles.css, it seems that the file changed while you was making these new changes, and now github does not know what to do, so you have to let it what to do. but don't worry it's easy fix, you can do that clicking on "resolve conflicts". let me know if you need help with that.

**Review by Enqira — APPROVED** [review]
The conflit was solved. some checks have failed but I think those are not related to this PR

**Review by dpfernandes — APPROVED** [review]
👍

**Pull request merged**